instruments/tests/test_tektronix/test_tektronix_tds224.py

```diff
@@ -6,18 +6,57 @@
 
 # IMPORTS ####################################################################
 
+from enum import Enum
+import time
+
+from hypothesis import given, strategies as st
 import numpy as np
+import pytest
 
 import instruments as ik
 from instruments.tests import expected_protocol, make_name_test
 
 # TESTS ######################################################################
 
-# pylint: disable=protected-access
+# pylint: disable=protected-access,redefined-outer-name
+
+
+# FIXTURES #
+
+
+@pytest.fixture(autouse=True)
```

Comment:
neat

Reply:
I think mocking out `time.sleep` this way, as you proposed some PRs ago is much nicer than using `if not self._testing`... And the autouse applies the mock to every test locally, if you need to check `assert_called` or something similar, you still have to pass it. `pytest` is great